SparkPost/2web2ui
# FE-419 Use Prettier + Stylelint for code formatting. Update ESLint.

### What Changed
 - Added the prettier code formatter
 - Added the stylelint style formatter
 - Update ESLint
 - Add plugins to turn off ESLint's code formatting rules
 - Add plugins to turn off Prettier's style formatting rules

### How To Test
 - Turn on format on save in VSCode (or other IDE, might have to google how to do so, but ensure that "Prettier" is in charge of it.
 - Save a js/jsx file that you know has formatting issues / change a file to have a bad format (add a string with double quotes or something). Save it, ensure the file is properly formatted.
 - Do the same with a scss file.
 - Turn off format on save, repeat the process and make a dummy branch (git checkout -b dummy-branch), commit the unformatted code, and make sure it gets formatted.
